Testing Niagara / Added Foostep VFX folder

Culltayne opened the pull request:
"Niagara Footstep VFX"

Culltayne committed:
Testing Niagara / Added Foostep VFX folder

"Niagara Footstep VFX"

brianisbrilliant commented:
Conflicting files
Content/Take3/dancer-take3.uasset
Content/Take3/dancer-take3_Skeleton.uasset 

I have made a folder with a copy of these assets so that your changes don't affect anyone else. you'll have to make the changes again because I am going to discard the changes you made to the models that we all share.

brianisbrilliant committed:
Updated CayneMap

created a copy of the Take3 dancer for Cayne, replaced the model in cayne's map, and put the laser pointer texture on there. I don't know what he changed with the animation, so he'll have to re-do those actions. Sorry! We don't want to be changing files that other people may be using.
